portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,40 +32,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> messages;
   private static final String COMMENT = "comment-input";
   private static final String NAME = "name-input";
-
-  @Override
-  public void init() {
-    messages = new ArrayList<>();
-  }
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final int maxResults = 10;
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Construct comment from user request.
     String fullComment = request.getParameter(NAME) + ": " + request.getParameter(COMMENT);
-    messages.add(fullComment);
 
     response.setContentType("text/html;");
     response.getWriter().println(fullComment);
 
+    Entity commEntity = new Entity("Comment");
+    commEntity.setProperty("content", fullComment);
```

Comment:
Instead of setting the full comment here as the content, you might want to consider storing the name and comment as two separate properties, which allows you to format the data however you want.